hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/ContainerStateMachine.java

```diff
@@ -265,6 +268,23 @@ public ContainerStateMachine(HddsDatanodeService hddsDatanodeService, RaftGroupI
 
   }
 
+  private void validatePeers(RaftServer server, RaftGroupId id) throws IOException {
```

Comment:
Do not pass member fields/methods.   Get them inside `validatePeers`.

Reply:
done